velox/expression/tests/ExprTest.cpp

```diff
@@ -4942,6 +4942,30 @@ TEST_F(ExprTest, disableMemoization) {
   ASSERT_EQ(stats["plus"].numProcessedRows, 3 * flatSize);
 }
 
+TEST_F(
+    ExprTest,
```

Comment:
Since the change is limited to DictionaryVector and the effects can be easily testes via its API, testing it via expression eval would be an overkill. I would recommend writing a test in VectorTest instead